### ISSUE TITLE
Alignment problem

### ISSUE THREAD

**Issue by SDGArmenia** [open-sdg/open-sdg#879]
How can I change the alignment of targets and indicators? Example 2.c։ https://armstat.github.io/sdg-site-armenia/2/

**Comment by brockfanning** [comment]
@SDGArmenia Can you describe what your goal is, maybe with a mockup?

A more broad change would be choose a different layout. The available layouts are `goal`, `goal-by-target`, and `goal-by-target-vertical`. This is controlled in the [goal pages](https://github.com/armstat/sdg-site-armenia/tree/develop/_goals).

For more control, you would probably next turn to CSS changes, which could go into [this file](https://github.com/armstat/sdg-site-armenia/blob/develop/assets/css/custom.scss).

**Comment by SDGArmenia** [comment]
All the targets and indicators are correctly arranged, only 2.c are in the wrong position.
What changes do I need to make in CSS to change only the position of 2.c?

**Comment by brockfanning** [comment]
@SDGArmenia Thanks for your patience. Apologies, I suggested "CSS" based on my impression that this was an "alignment" issue, but I see now that it is a "sorting" issue. I don't understand why that 2.c target is out of order, to be honest. It is very odd. I know that we have made some improvements to the indicator sorting. Unfortunately you are using an older version of Open SDG, so you would need to upgrade in order to benefit from those improvements.
